modules/library/src/main/scala/zio/elasticsearch/query/Queries.scala

```diff
@@ -628,9 +626,15 @@ sealed trait MultiMatchQuery[S]
    * [[zio.elasticsearch.query.ElasticQuery]] is executed internally.
    *
    * @param matchingType
-   *   the [[zio.elasticsearch.query.MultiMatchType]] value of 'type' parameter
+   *   the [[zio.elasticsearch.query.MultiMatchType]] value of 'type' parameter, possible values are:
+   *   - [[zio.elasticsearch.query.MultiMatchType.BestFields]]
+   *   - [[zio.elasticsearch.query.MultiMatchType.BoolPrefix]]
+   *   - [[zio.elasticsearch.query.MultiMatchType.CrossFields]]
+   *   - [[zio.elasticsearch.query.MultiMatchType.MostFields]]
+   *   - [[zio.elasticsearch.query.MultiMatchType.Phrase]]
+   *   - [[zio.elasticsearch.query.MultiMatchType.PhrasePrefix]]
```

Comment:
Please, provide short explanation for each value. You can see `scaladoc` for `scoreMode` def.